Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM mhart/alpine-node
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/
 
+RUN apk --update add git build-base && \
+    apk del build-base && rm -fr /usr/share/ri
```

Comment:
That example needs `build-base` present for the `bundle` step in the middle. This doesn't seem to fit that.